NineChronicles.Headless/GraphTypes/StandaloneMutation.cs

```diff
@@ -85,6 +85,76 @@ public StandaloneMutation(StandaloneContext standaloneContext, IConfiguration co
             ).AuthorizeWith(GraphQLService.UserPolicyKey);
 
             Field<TxIdType>(
+                name: "transfer",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "A hex-encoded value for address of recipient.",
+                        Name = "recipient",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Description = "A string value of the value to be transferred.",
+                        Name = "amount",
+                    },
+                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    {
+                        Description = "A sender's transaction counter. You can get it through nextTxNonce().",
+                        Name = "txNonce",
+                    },
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "A hex-encoded value for address of currency to be transferred. The default is the NCG's address.",
+                        DefaultValue = GoldCurrencyState.Address,
+                        Name = "currencyAddress"
+                    }
+                ),
+                resolve: context =>
+                {
+                    if (!(standaloneContext.NineChroniclesNodeService is { } service))
+                    {
+                        throw new InvalidOperationException($"{nameof(NineChroniclesNodeService)} is null.");
+                    }
+
```

Comment:
Since #420 merged, in my thought, this check seems not needed 🤔

Reply:
🆗

Reply:
On second thought, `standaloneContext.NineChroniclesNodeService` is still nullable and this check can be useful. (or, we need to change this field to non-nullable.)